back-end/e-learning/app/Models/Category.php

```diff
@@ -18,4 +20,9 @@ class Category extends Model
 	protected $dates = [
 		'deleted_at',
 	];
+
+	public function word()
```

Comment:
changes this to `words` here because the relationship of a category to a word is more than one. Although this does not affect functionality, it's still important to make sure that code is readable at all times.